src/components/Ballot.test.tsx

```diff
@@ -43,8 +46,22 @@ it(`can navigate all ballot pages`, () => {
 })
 
 it('redirects contests index to first contest', () => {
-  const { container } = render(<Ballot />, {
+  const { container, getByText } = render(<Ballot />, {
     route: '/contests',
   })
+  getByText('President')
+  expect(container.firstChild).toMatchSnapshot()
+})
+
+it('skips activation if when disabled', () => {
+  const { container, getByText } = render(<Ballot />, {
+    election: lodashMerge(electionSample, {
```

Comment:
being pedantic, I'd make defaults an optional argument to `mergeWithDefaults` and use it here. But this is optional, not required.